LinearRegression/LinearRegressionBackend/MLNeuralNetwork/NeuralNetwork.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+using MathNet.Numerics.LinearAlgebra;
+
+namespace LinearRegressionBackend.MLNeuralNetwork
+{
+    public class NeuralNetwork
+    {
+        private List<Layer> _Layers;
+
+        public NeuralNetwork(List<Layer> layers)
+        {
+            _Layers = layers;
+        }
+
+        public Vector<double> PropagateForward(Vector<double> inputData)
+        {
+            Vector<double> activations = inputData;
+
+            foreach (Layer layer in _Layers)
+            {
+                activations = layer.Activations(activations);
```

Comment:
A = inputData
for i .. layer
   weightedSum = Layer.WeightedSums(A)
   A = Layer.Activation(weightedSum)

Reply:
The calculation is fixed by the upcoming commit.